pom.xml

```diff
@@ -183,7 +184,27 @@
                     </dependency>
                 </dependencies>
             </plugin>
-
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <version>3.0.0-M2</version>
```

Comment:
Should this be `<version>3.0.0-M3</version>`?

Reply:
Actually the final version 3.0.0 of the Maven Enforcer Plugin [was recently released](https://github.com/apache/maven-enforcer/commit/b1b22822174bc92857a2e674c9a024035ee6d7cd), so we should use that.